source

```diff
@@ -8779,9 +8814,8 @@ a.setAttribute('href', 'https://example.com/'); // change the content attribute
   consistency.</p>
 
   <p>If a <span>reflected IDL attribute</span> has the type <code
-  data-x="">FrozenArray&lt;<var>T</var>&gt;?</code>, where <var>T</var> is either
-  <code>Element</code> or an interface that inherits from <code>Element</code>, then with
-  <var>attr</var> being the <span>reflected content attribute name</span>:</p>
+  data-x="">FrozenArray&lt;Element&gt;?</code>, then with <var>attr</var> being the <span>reflected
+  content attribute name</span>:</p>
```

Comment:
This is a bit unsatisfying, but with reference target we can't really enforce typing any more.